.github/workflows/test.yml

```diff
@@ -12,13 +12,19 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ ubuntu-20.04, macOS-latest, windows-latest ]
```

Comment:
We previously had a discussion whether to tests only on the latest Ubuntu (see https://github.com/audeering/audobject/pull/35#discussion_r826801892). So I wonder what your motivation was to change it here?

Reply:
I had a second thought on this.
Most of the time I used `ubuntu-20.04` vs. `ubuntu-latest` to use only Python 3.8 with `ubuntu-20.04` to only build the documentation there. Now that we have the `tasks` this is no longer needed, and I think it make sense to only use `ubuntu-latest` as this is also easier to maintain over time as we don't need to change it, whereas `ubuntu-20.04` will no longer be available in a couple of years.